components/mosart/src/wrm/WRM_subw_IO_mod.F90

```diff
@@ -448,7 +448,7 @@ subroutine WRM_init
               enddo
            enddo
            if (cnt /= cntg) then
-              write(iulog,"(a,3i8)"), subname//'ERROR: sMat d2g cnt errorb',cntg,cnt
+              write(iulog,"(a,3i8)") subname//'ERROR: sMat d2g cnt errorb',cntg,cnt
```

Comment:
`trim(subname)`

Reply:
yes, it is always better to use "trim(subname)" instead of "subname", I will do it later.